Arrotech/API_ENDPOINTS
# #161770410 Create a delivery order

**_What does this PR do?_**
A user can create a parcel delivery order.

**_Description of Task to be completed?_**
Create a view for the parcel delivery order.
Create a model for the parcel delivery order.

**_How should this be manually tested?_**
This should be tested using the postman app. A user should run the url specified to create an order. This should be run as POST

**_Any background context you want to provide?_**
A user should manually create an order in the postman app to test if the POST method work.


**_What are the relevant pivotal tracker stories?_**
#161770410
